*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:45829 (IPv4)
Extension host agent listening on 45829

[04:24:48] Extension host agent started.
[04:24:49] [127.0.0.1][6f650166][ManagementConnection] Unknown reconnection token (never seen).
[04:24:49] [127.0.0.1][01964e99][ExtensionHostConnection] Unknown reconnection token (never seen).
[09:10:19] [127.0.0.1][5ca672bb][ManagementConnection] New connection established.
[09:10:19] [127.0.0.1][b683646b][ExtensionHostConnection] New connection established.
[09:10:19] [127.0.0.1][b683646b][ExtensionHostConnection] <29373> Launched Extension Host Process.
rejected promise not handled within 1 second: Error: Could not find pty on pty host
stack trace: Error: Could not find pty on pty host,    at m._throwIfNoPty (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at m.updateTitle (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:1673),    at Object.call (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:13:8361),    at g.onPromise (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4879),    at g.onRawMessage (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4267),    at /home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:3555,    at D.invoke (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at N.deliver (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at h.fire (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.p (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:6:34973),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)
[09:10:21] [Error: Could not find pty on pty host,    at m._throwIfNoPty (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:5333),    at m.updateTitle (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:20:1673),    at Object.call (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:13:8361),    at g.onPromise (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4879),    at g.onRawMessage (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:4267),    at /home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:12:3555,    at D.invoke (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:145),    at N.deliver (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:2271),    at h.fire (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:8:1849),    at process.p (/home/rauli/.vscode-server/bin/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/platform/terminal/node/ptyHostMain.js:6:34973),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)]
[09:50:51] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /home/rauli)
[09:50:51] [File Watcher (universal)] restarting watcher after error: std::bad_alloc
[09:50:51] [File Watcher (node.js)] Watcher shutdown because watched path got deleted
